New common 'VS Code Troubleshooting' page

lhelms2 opened the pull request:
Adds a new common 'VS Code Troubleshooting' page. Page include:

- 'Cannot Log in with VS Code - Disk Quota Exceeded'
- 'VS Code Performance Issues' (pulled from Delta docs)

I also added a link to this new page at the top of the 'Quotas' page in case a user ends up there for the VS Code disk quota exceeded issues.

RTD build: https://docs.ncsa.illinois.edu/en/vscode-troubleshoot/common/vscode.html

pmenstrom reviewed (approved):
Looks good, I did check the links.

lhelms2 commented:
Re-worked/expanded page based on Greg's request.

- Renamed to 'VS Code on NCSA Resources'
- Added Use Policy section
- Added System-Specific Information section

RTD build: https://docs.ncsa.illinois.edu/en/vscode-troubleshoot/common/vscode.html

pmenstrom reviewed (approved):
I clecked all of the links and they look good except the HAL VS Code link goes to Delta AI.

lhelms2 commented:
thank you @pmenstrom , I fixed the HAL link
